orchestrator_builder.sh

```diff
@@ -112,7 +119,7 @@ get_sources(){
     cp ${WORKDIR}/orchestrator-packaging/percona-orchestrator.spec ./
     cd ../
     mv ${WORKDIR}/orchestrator-packaging/debian ./
-    ver=$(git rev-parse HEAD)
+    ver="${VERSION}-($(git rev-parse --short HEAD))"
```

Comment:
Can we add the RELEASE here? So it prints something like 3.2.6-3, where 3 is the RELEASE number in this case. Then we can have a space, and then the parenthesis. E.g. 3.2.6-3 (a48d8b2). If we do not add the RELEASE number the confusion will persist. Thanks

Reply:
Thank you Santo. It will be changed by another PR.
We have got a [separate ticket](https://jira.percona.com/browse/DISTMYSQL-168) that is related to versions.